src/main/java/org/sonar/plugins/checkstyle/metadata/CheckstyleMetadata.java

```diff
@@ -191,6 +262,12 @@ private static String getRuleTag(String checkPackage,
         return result;
     }
 
+    /**
+     * Fetch additional module details from a YML config.
+     *
+     * @param fileName YML config file
+     * @return map of additional metadata
+     */
```

Comment:
move javadocs to separate commit, it not functional change.
(apply same to other code)

Reply:
Done in new reorganized PR https://github.com/checkstyle/sonar-checkstyle/pull/306